src/core/Helpers/Filesystem.cpp

```diff
@@ -129,6 +129,8 @@ bool Filesystem::bootstrap( Logger* logger, const QString& sys_path )
 #ifdef H2CORE_HAVE_BUNDLE
 	// Bundle: Prepare hydrogen to use path names which are used in app bundles: http://en.wikipedia.org/wiki/Application_Bundle
 	__sys_data_path = QCoreApplication::applicationDirPath().append( "/../Resources/data/" ) ;
+#elif H2CORE_HAVE_APPIMAGE
```

Comment:
You use Debian in a Docker image, aren't you?

This `#ifdef` is placed within the `#ifdef Q_OS_MACX` clause. It will only be executed if compiled on macOS. You need to move it to the Linux section starting at line 143.

Reply:
That did it, thanks! I have a working AppImage.

However, now I broke my PR. I'll create a new one and put it up.